<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,2">
       <item>
        <widget class="QLineEdit" name="lineEdit_addr"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_disconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_video_display">
        <attribute name="title">
         <string>Display Window</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_original_display">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>350</height>
              </size>
             </property>
             <property name="title">
              <string>Original</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="video_original">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_yolo_display">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>350</height>
              </size>
             </property>
             <property name="title">
              <string>YOLO Display</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="video_yolo">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_midas_display">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>350</height>
              </size>
             </property>
             <property name="title">
              <string>MiDas Display</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QLabel" name="video_midas">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QGroupBox" name="groupBox_yolo_config">
                 <property name="title">
                  <string>YOLO</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_3">
                      <item>
                       <widget class="QRadioButton" name="radioButton_yolov1">
                        <property name="text">
                         <string>YOLOv1</string>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QRadioButton" name="radioButton_yolov2">
                        <property name="text">
                         <string>YOLOv2</string>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QRadioButton" name="radioButton_yolov3">
                        <property name="text">
                         <string>YOLOv3</string>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QRadioButton" name="radioButton_yolov4">
                        <property name="text">
                         <string>YOLOv4</string>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="Line" name="line">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_4">
                      <item>
                       <widget class="QRadioButton" name="radioButton_yolov5">
                        <property name="text">
                         <string>YOLOv5</string>
                        </property>
                        <property name="checked">
                         <bool>true</bool>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QRadioButton" name="radioButton_yolov6">
                        <property name="text">
                         <string>YOLOv6</string>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QRadioButton" name="radioButton_yolov7">
                        <property name="text">
                         <string>YOLOv7</string>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QRadioButton" name="radioButton_yolov8">
                        <property name="text">
                         <string>YOLOv8</string>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>17</width>
                      <height>13</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_midas_config">
                 <property name="title">
                  <string>MiDaS</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QRadioButton" name="radioButton_midas_small">
                    <property name="text">
                     <string>MiDaSv2.1-Small</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_midas_hybrid">
                    <property name="text">
                     <string>MiDaSv3-Hybrid</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_midas_large">
                    <property name="text">
                     <string>MiDaSv3-Large</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="pushButton_loadmodel">
                 <property name="text">
                  <string>Load Model</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_launch">
                 <property name="text">
                  <string>Launch</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_stop">
                 <property name="text">
                  <string>Stop</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
